client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -333,8 +336,44 @@ class SimplePaymentsDialog extends Component {
 		return buttons;
 	}
 
+	renderEmptyDialog( content ) {
+		const { onClose, translate, showDialog } = this.props;
+		return (
+			<Dialog
+				isVisible={ showDialog }
+				onClose={ onClose }
+				buttons={ [
+					<Button onClick={ onClose }>
+						{ translate( 'Cancel' ) }
+					</Button>,
+				] }
+				additionalClassNames="editor-simple-payments-modal"
+			>
+				<Navigation
+					activeTab={ 'list' }
+					paymentButtons={ [] }
+					onChangeTabs={ noop }
+				/>
+				{ content }
+			</Dialog>
+		);
+	}
+
+	returnTrue() {
```

Comment:
Very funny method :) `UpgradeNudge` should be changed to allow both function and boolean as the value of `shouldDisplay`.